packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -194,8 +194,8 @@ module.exports = {
             link: 'https://webpack.js.org/concepts/hot-module-replacement/',
         },
         {
-            name: 'sourcemap',
-            usage: '--sourcemap <sourcemap | eval>',
+            name: 'devtool',
+            usage: '--devtool <source-map | eval>',
```

Comment:
need fix it, because we have more values here

Reply:
done @evilebottnawi 👍